definitions/divorce/json/AuthorisationCaseField.json

```diff
@@ -12634,160 +12634,6 @@
     "UserRole": "caseworker-divorce-solicitor",
     "CRUD": "R"
   },
-  {
```

Comment:
@qzhou-hmcts do we have to feature flag some code or tests as we're reverting these?

Reply:
COS already has the feature flag